components/layout_2020/table/mod.rs

```diff
@@ -272,12 +276,13 @@ impl TableSlot {
 }
 
 /// A row or column of a table.
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, MallocSizeOf)]
 pub struct TableTrack {
     /// The [`BaseFragmentInfo`] of this cell.
     base_fragment_info: BaseFragmentInfo,
 
     /// The style of this table column.
+    #[ignore_malloc_size_of = "stylo type"]
```

Comment:
For this Stylo type and all of the others please add a `malloc_size_of_is_stylo_malloc_size_of` line to `components/malloc_size_of/lib.rs` (if Stylo has its own `MallocSizeOf` implementation) or use `malloc_size_of_is_0` there instead of ignoring in each data structure. Thanks!